### PR TITLE
add styling to codebuild output

### PR DESCRIPTION
**Pull request by ejayesh**
Update codebuild_cli to have similar output styling to dns_cli

### CODE REVIEW

**Review by WillGibson — COMMENTED** [review]
Oh, nice. I'm going to play with colours now in my branch!

**Review by WillGibson — APPROVED** [review]

File: commands/codebuild_cli.py [hunk 1]
```diff
@@ -5,9 +5,8 @@
 import click
 from boto3.session import Session
 from mypy_boto3_codebuild.client import CodeBuildClient
-
-from utils.aws import check_aws_conn
-from utils.aws import check_response
+from .utils import check_aws_conn
+from .utils import check_response
```
Comment (yusufsheiqh) [inline commands/codebuild_cli.py:9]: I thought we wanted to move everything out to the `utils` folder. 👀

File: commands/codebuild_cli.py [hunk 2]
```diff
@@ -21,15 +20,15 @@ def import_pat(pat: str, client: CodeBuildClient):
...
+    click.secho("PAT successfully added", fg="green")
 
 
 def check_github_conn(client: CodeBuildClient):
     response = client.list_source_credentials()
 
     # If there are no source code creds defined then AWS is not linked to Github
     if not response["sourceCredentialsInfos"]:
-        if not click.confirm("Github not Linked in this AWS account\nDo you want to link with a PAT?"):
+        if not click.confirm(click.style("Github not Linked in this AWS account\nDo you want to link with a PAT?", fg="yellow")):
```
Comment (yusufsheiqh) [inline commands/codebuild_cli.py:31]:
```suggestion
        if not click.confirm(click.style("GitHub is not linked in this AWS account\nDo you want to link with a PAT?", fg="yellow")):
```

File: commands/codebuild_cli.py [hunk 3]
```diff
@@ -53,7 +52,7 @@ def check_service_role(project_session: Session) -> str:
         role_arn = response["Role"]["Arn"]
 
     except client.exceptions.NoSuchEntityException:
-        print("Role for service does not exist run ./codebuild_cli.py create-codeploy-role")
+        click.secho("Role for service does not exist run ./codebuild_cli.py create-codeploy-role", fg="cyan")
```
Comment (yusufsheiqh) [inline commands/codebuild_cli.py:55]:
```suggestion
        click.secho("Role for service does not exist; run ./codebuild_cli.py create-codeploy-role", fg="cyan")
```

File: commands/codebuild_cli.py [hunk 4]
```diff
@@ -79,19 +78,15 @@ def check_git_url(git: str) -> str:
...
     else:
-        print(
-            """
-            Unable to recognise git url format, make sure its either:
-            https://github.com/<org>/<repository-name>
-            git@github.com:<org>/<repository-name>
-            """,
-        )
+        click.echo(
+            click.style("Unable to recognise git url format, make sure its either:\n", fg="red") +
```
Comment (yusufsheiqh) [inline commands/codebuild_cli.py:87]:
```suggestion
            click.style("Unable to recognise Git URL format, make sure it is either:\n", fg="red") +
```

File: commands/codebuild_cli.py [hunk 5]
```diff
@@ -235,7 +230,7 @@ def codedeploy(update, name, desc, git, branch, buildspec, builderimage, project
                 logsConfig=logsConfig,
             )
         except client.exceptions.ResourceNotFoundException:
-            print("Unable to update a project that does not exist, drop the --update flag")
+            click.secho("Unable to update a project that does not exist, drop the --update flag", fg="red")
```
Comment (yusufsheiqh) [inline commands/codebuild_cli.py:233]:
```suggestion
            click.secho("Unable to update a project that does not exist, remove the --update flag", fg="red")
```

**Review by yusufsheiqh — APPROVED** [review]

File: commands/codebuild_cli.py (hunk 1 above)
Comment (ejayesh) [reply commands/codebuild_cli.py:9]: i did, but Lyndon created a utils file.  I didnt want utils in two palces

Comment (yusufsheiqh) [reply commands/codebuild_cli.py:9]: okie dokie

**Pull request merged**